frontend/src/metabase/admin/people/containers/UserSuccessModal.jsx

```diff
@@ -22,6 +23,9 @@ class UserSuccessModal extends React.Component {
   }
   render() {
     const { onClose, user, temporaryPassword } = this.props;
+    const isEeSsoConfigured =
+      MetabaseSettings.ssoConfigured() &&
+      !MetabaseSettings.passwordLoginEnabled();
```

Comment:
Use the same logic as in BE
https://github.com/metabase/metabase/blob/1cdf0e646ad8a34cc954760affc52c6fab54d9e2/src/metabase/models/user.clj#L247-L248